pages/faq/troubleshooting/faq.md

```diff
@@ -5,29 +5,30 @@ title: FAQs
+ - [Can I access /dev and things like GPIO from the container?](#can-i-access-dev-and-things-like-gpio-from-the-container)
+ - [Can I set a static IP address for my device?](#can-i-set-a-static-ip-address-for-my-device)
+ - [Why can't I SSH into or run code in older versions of the host OS?](#why-cant-i-ssh-into-or-run-code-in-older-versions-of-the-host-os)
+ - [Which data is persisted on devices across updates/power cycles?](#which-data-is-persisted-on-devices-across-updatespower-cycles)
+ - [Why does /data disappear when I move a device between fleets?](#why-does-data-disappear-when-i-move-a-device-between-fleets)
+ - [It appears that there is a centralized master running (in cloud) and agents running on devices. Is that accurate?](#it-appears-that-there-is-a-centralized-master-running-in-cloud-and-agents-running-on-devices-is-that-accurate)
+ - [What type of encryption do you use over OpenVPN? SSL/TLS/AES-256? Mutual key authentication? over SSH?](#what-type-of-encryption-do-you-use-over-openvpn-ssltlsaes-256-mutual-key-authentication-over-ssh)
+ - [What is the performance impact on the gateway device due to encryption?](#what-is-the-performance-impact-on-the-gateway-device-due-to-encryption)
+ - [How long does the update process run typically? Do you have any benchmark data? For now it appears to be quick for small updates.](#how-long-does-the-update-process-run-typically-do-you-have-any-benchmark-data-for-now-it-appears-to-be-quick-for-small-updates)
+ - [How does the device registration work over the VPN and how do you ensure the identity of the device on the first-time registration?](#how-does-the-device-registration-work-over-the-vpn-and-how-do-you-ensure-the-identity-of-the-device-on-the-first-time-registration)
+ - [If the device is installed behind a proxy/firewall and can’t be reachable on internet via direct connection, what are the pitfalls?](#if-the-device-is-installed-behind-a-proxyfirewall-and-cant-be-reachable-on-internet-via-direct-connection-what-are-the-pitfalls)
+ - [How do you secure your own cloud to prevent malicious attack which may allow attacker to break-in to our systems?](#how-do-you-secure-your-own-cloud-to-prevent-malicious-attack-which-may-allow-attacker-to-break-in-to-our-systems)
+ - [When are device types discontinued?](#when-are-device-types-discontinued)
+ - [What does it mean when a device type is discontinued?](#what-does-it-mean-when-a-device-type-is-discontinued)
+ - [I have a device that is not on the supported devices list. Can it run on {{ $names.company.lower }}?](#i-have-a-device-that-is-not-on-the-supported-devices-list-can-it-run-on--namescompanylower-)
+ - [What to keep in mind when choosing power supply units?](#what-to-keep-in-mind-when-choosing-power-supply-units)
+ - [Does {{ $names.company.lower }} have access to my device, source code and images?](#does--namescompanylower--have-access-to-my-device-source-code-and-images)
 
```

Comment:
Your editor auto indented this, we need the index to show this as table of contents. Please do revert this

Reply:
You are indeed correct, thanks for fixing this. I don't know when exactly this broke. 
Preview here: https://ce23a6e8.balenacloud-docs.pages.dev/faq/troubleshooting/faq/